creme/creme_core/templates/creme_core/templatetags/listview/entity-filters.html

```diff
@@ -30,9 +30,9 @@
     </div>
  {% with ctype=model|ctype_for_instance %}
   {% if selected != None %}
-    <a data-action="popover">
+    <a data-action="popover" data-title="{% blocktranslate %}Conditions of «{{selected}}»{% endblocktranslate %}">
```

Comment:
`{% blocktranslate with efilter=selected %}Conditions of «{{efilter}}»{% endblocktranslate %}`
so we do not have to create a new 'msgid' in translation file